crates/cext/src/dag.rs

```diff
@@ -841,6 +1004,41 @@ pub unsafe extern "C" fn qk_dag_op_node_kind(dag: *const DAGCircuit, node: u32)
     }
 }
 
+/// @ingroup QkDag
+/// Return the details for an instruction in the circuit.
+///
+/// This is a mirror of ``qk_circuit_get_instruction``.  You can also use individual methods
+/// such as :c:func:`qk_dag_op_node_gate_op` to get individual properties.
```

Comment:
```suggestion
/// such as ``qk_dag_op_node_gate_op`` to get individual properties.
```
As mentioned before, due to some strange behavior, this format doesn't link correctly. A double tick should suffice.

Reply:
Thanks, yeah - I think the cross-refs are because I wrote this before the other PR that we talked about.

Fwiw, there's no need for doubling the ticks for the same reason that the cross-reference doesn't work - this gets pre-processed by Doxygen (as Markdown) which only needs single backticks (but double should be harmless).

Fixed in 6c36a89c41ca9b74d38301edff4dbf6aea292cf0.